index.html

```diff
@@ -9,24 +9,21 @@
     </title>
     <meta name="description" content="A new community for blockchain developer, designers and Dapp Owners" />
     <meta name="keywords" content="" />
-    <meta name="author" content="mkubdev" />
+    <meta name="author" content="@github:mkubdev" />
     <link rel="stylesheet" href="https://unpkg.com/tailwindcss/dist/tailwind.min.css" />
     <!--Replace with our tailwind.css once created, we use the base tailwind atm-->
     <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,700" rel="stylesheet" />
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
-    <link rel="icon" href="favicon.ico" type="image/x-icon">
+    <link rel="icon" href="./assets/favicon.ico" type="image/x-icon">
```

Comment:
Wouldn't this be the correct code?
```suggestion
    <link rel="icon" href="/assets/favicon.ico" type="image/x-icon">
```

Reply:
fixed!